src/screens/TicketManagement/MyTicketComponent.jsx

```diff
@@ -274,12 +277,39 @@ export default function MyTicketComponent() {
                 </Link>{" "}
               </li>
 
-              {data.created_by != localStorage.getItem("id") &&
+              {/* {data.created_by != localStorage.getItem("id") &&
                 data.basket_configured === 0 &&
                 localStorage.getItem("account_for") === "SELF" &&
                 data.status_name != "Solved" &&
                 data.passed_status !== "REJECT" &&
-                data.passed_status !== "UNPASS" && (
+                data.passed_status !== "UNPASS" && ( */}
```

Comment:
if it's not req in the feature remove commented code,
 if req add comment in the code